fabric/build.gradle

```diff
@@ -24,6 +24,16 @@ loom {
     accessWidenerPath = project(":common").loom.accessWidenerPath
 }
 
+repositories {
```

Comment:
Remove the extra repositories before merging. Our CI should automatically pull the necessary artifacts into our maven